CHANGELOG.md

```diff
@@ -10,12 +10,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.12.0][v0.12.0_log]
+--------------
+
 ### Added
 - Initial implementation of the [Malloc Info API](docs/DesignDocs/Mallinfo.md) for dlmalloc (default allocator), and snmalloc.
 - Added missing attribute validations to oeedger8r C++ implementation.
+- Added new API *oe_log_message* see [design doc](docs/DesignDocs/oe_log_message()_callback_proposal.md) and [sample](samples/log_callback/README.md).
+
+### Changed
+- Fixed #3543, updated openenclaverc file and documents on Windows to avoid overwriting CMAKE_PREFIX_PATH.
```

Comment:
```suggestion
- Fixed https://github.com/openenclave/openenclave/issues/3543, updated openenclaverc file and documents on Windows to avoid overwriting CMAKE_PREFIX_PATH.
```

Once cloned locally, I believe the markdown won't automatically recognize issue numbers so any number should have a link, like lines 31, 35, etc do.